### PR TITLE
Set max in and out bound - 1

### PR DESCRIPTION
**Pull request by minsulee2**

### CODE REVIEW

File: common/constants.js [hunk 1]
```diff
@@ -467,7 +467,7 @@ function initializeNetworkEnvronments() {
   } else {
     return {
       // Note(minsu): Need a discussion that MIN_NUM_VALIDATORS and MAX_INBOUND_LIMIT should not be
-      // related one another.
+      // related to one another.
       MAX_OUTBOUND_LIMIT: GenesisParams.consensus.MIN_NUM_VALIDATORS - 1,
       MAX_INBOUND_LIMIT: GenesisParams.consensus.MIN_NUM_VALIDATORS - 1,
```
Comment (minsulee2) [inline common/constants.js:472]: Sorry for confusing. I directly pushed to develop with -1 parts by mistake.  I made another branch and PR for this issue #348

Comment (platfowner) [reply common/constants.js:472]: I think that shouldn't be possible under the current branch configuration(i.e., requiring at least 1 approval)?:
https://github.com/ainblockchain/ain-blockchain/settings/branches

Comment (minsulee2) [reply common/constants.js:472]: I thought so too. But it happened (I still don't get it too). https://github.com/ainblockchain/ain-blockchain/blob/develop/common/constants.js#L464

Comment (platfowner) [reply common/constants.js:472]: Just confirmed that our branch config is incomplete. I'll take a look. :)

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

File: common/constants.js (hunk 1 above)
Comment (minsulee2) [reply common/constants.js:472]: Thanks! I am glad that found a security hole by accident :)

**Pull request merged**